examples/network/http_server_simplistic.py

```diff
@@ -28,7 +28,7 @@ def main():
     while True:
         res = s.accept()
         client_s = res[0]
-        client_addr = res[1]
```

Comment:
Does this mean that there is a convention in Python land that names beginning with underscores mean 'unused'? I though it was 'private', which is not the same at all. Wouldn't using `# noqa: unused` or so be better here? That doesn't need to mangle the name and makes it very explicit and clear that this variable is here only to show what res[1] is.

Reply:
Please reread the commit message.

https://dbader.org/blog/meaning-of-underscores-in-python

Reply:
That doesn't mention unused at all, only private?

Also what do you think about the other suggestions?

Reply:
Given that this local variable is never used again, this `_` indicates that it is considered private (to this line alone).
* https://github.com/PyCQA/pyflakes/issues/393 from 2018.

The `# noqa: F841` but clunky when reading the code most contributors will not have ruff's 500 rules memorized.

My hope is that when maintainers review this pull request, they spot the logic error and remove the whole line it that makes sense to them.  For me, it would be a SWAG because I do not know the codebase.

It is interesting to note that `ruff --select=F841 --fix .` is more brutal than am.
* `client_addr = res[1]` --> `res[1]`  # Which is clearly a noop.

To which flake-bugbear would then reply:
> B018: Found useless expression. Either assign it to a variable or remove it.

Reply:
>  and remove the whole line 

That would actually be my preference as well

> Who cares if the name is mangled if it never gets used?

The reader. The line not being there means less reading.

Reply:
I can close this PR and you can create a replacement if you want.  `ruff --select=F841 --fix .` is easy to run.

Reply:
I'd say just remove this line.

This is an example and is actually a simplified version of `http_server_simplistic_commented.py`.  The latter uses this variable, so if someone wants more details they should look at `http_server_simplistic_commented.py` instead.